examples/Cassie/run_simple_sim.cc

```diff
@@ -228,6 +233,13 @@ int do_main(int argc, char* argv[]) {
 
   std::cout << q0 << std::endl;
 
+  // If it's floating base and the initial height is 0, we adjust it to be above
+  // the ground.
+  if (FLAGS_floating_base){
+    if (x0(2) == 0)
+      x0(2) = 1.5;
```

Comment:
This is a strange sequence. Why `if x0(2) == 0`? What if it's close to zero? Why 1.5?

Reply:
The idea is to prevent someone in the future who wants to use `run_simple_sim.cc` with floating-base model but forgets to set the initial height of the robot

Reply:
1.5 is just a random selected number that guarantees the robot is above the ground, so when the simulation runs the robot is not below the ground

Reply:
I'm not sure if I buy the reasoning for this. The user needs to set the initial configuration, somehow. I could see something like
```
if (FLAGS_floating_base)
 x0(2) = 1.5
```
being used right after `x0` is first created.

Reply:
Yes, I can change to that